### DISCUSSION TITLE
"Human" Coverage format in "deploy" command

### DISCUSSION THREAD

**Discussion by madhav201098** [forcedotcom/cli#2512] [Ideas]
**Problem Statement:**
For Sonarqube to pass the coverage report, as of now we need to use the "**sf apex test run**" command to generate the report. We in our organization, use scratch org for our development. As part of CI, we have a check to validate the components created & running Unit tests as well which is increasing the CI time gradually. We are designing an approach in which we want to generate the coverage report as part of the "**deploy**" command itself instead of running a separate command to generate the coverage report. 

**Issue:**
We have identified a crucial aspect in the generation of a Sonar-accepted report, specifically the requirement to include the "**Human**" parameter when using the "**sf apex test run**" command. However, it's worth noting that this parameter is not currently supported within the "**deploy**" command.

**Feature Request:**
Could we explore the feasibility of introducing a new "Human" parameter within the "deploy" command? This parameter could streamline the process of generating the coverage report within the same command, thus eliminating the necessity of running a separate command for this purpose.

**Comment by mcarvin8** [comment]
If you need a way to get code coverage from either CLI command (deploy or test) into SonarQube, feel free to use my CLI plugin https://github.com/mcarvin8/apex-code-coverage-transformer

I've been using this with SonarQube for about a year now.
